docs/content/common-subs/create-ks-core-kind-cluster.md

```diff
@@ -0,0 +1,36 @@
+<!--create-ks-core-kind-cluster-start-->
+create the ks-core kind cluster
+```shell
+KUBECONFIG=~/.kube/config kind create cluster --name ks-core --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 443
+    hostPort: {{ config.ks_kind_port_num }}
```

Comment:
Do you mean to specify a port number here? If I run this verbatim I get:
`ERROR: failed to create cluster: unable to decode config: yaml: unmarshal errors:
  line 13: cannot unmarshal !!map into int32`